stubs/PyAutoGUI/pyautogui/__init__.pyi

```diff
@@ -1,37 +1,177 @@
 import contextlib
+from _typeshed import Incomplete
 from collections.abc import Callable, Generator, Iterable, Sequence
 from datetime import datetime
-from typing import NamedTuple, TypeVar, overload
-from typing_extensions import ParamSpec
+from typing import NamedTuple, SupportsFloat, SupportsInt, TypeVar, overload
+from typing_extensions import ParamSpec, SupportsIndex, TypeAlias
 
-# from pyscreeze import Box
+from PIL import Image
 
 class PyAutoGUIException(Exception): ...
 class FailSafeException(PyAutoGUIException): ...
 class ImageNotFoundException(PyAutoGUIException): ...
 
 _P = ParamSpec("_P")
 _R = TypeVar("_R")
+_NormalizeableXArg: TypeAlias = SupportsInt | Sequence[SupportsInt]
```

Comment:
We covered this in the last PR :)

The `_normalizeXYArgs` fnction explicitly accounts for the fact that the `x` argument might be a string: https://github.com/asweigart/pyautogui/blob/667161b072e89b4a25b806c5f2ca616e6c0aeec3/pyautogui/__init__.py#L664

Unfortunately, `SupportsInt` means "anything that has an `__int__` method, not `anything that can be passed to the `int()` constructor -- hence why we need to explicitly include `str` in this union.

```suggestion
_NormalizeableXArg: TypeAlias = str | SupportsInt | Sequence[SupportsInt]
```